client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -129,6 +129,14 @@ const BrewRenderer = createClass({
 			return <div className='phb3 page' id={`p${index + 1}`} dangerouslySetInnerHTML={{ __html: Markdown.render(pageText) }} key={index} />;
 	},
 
+	renderStyle : function(){
+		if(!this.props.brew.style) return;
+		if(this.props.renderer == 'legacy')
+			return <div style={{ display: 'none' }} dangerouslySetInnerHTML={{ __html: MarkdownLegacy.render(`<style> ${this.props.style} </style>`) }} />;
+		else
+			return <div style={{ display: 'none' }} dangerouslySetInnerHTML={{ __html: Markdown.render(`<style> ${this.props.style} </style>`) }} />;
```

Comment:
We can just render all the style panels using `Markdown.render` since being in Legacy or V3 mode shouldn't affect the way the Style panel is processed.